docs/en/appium-setup/running-on-windows.md

```diff
@@ -1,46 +1,27 @@
-## Running Appium on Windows
+﻿#Windows Setup
+
+Appium on Windows supports both Windows and Android app automation!
 
-### Limitations
+See [Windows App Testing](windows-app-testing.md) for more details.
 
-If you are running Appium on Windows, you can use the
-[Appium.exe](https://github.com/appium/appium-dot-exe) client, which will allow
-you to quickly launch an Appium server and use the Inspector. You will not be
-able to test iOS apps on a locally hosted server, because Appium relies on OS
-X-only libraries to support iOS testing. You can however use the `Remote Server`
-option to connect to an Appium server running on a Mac.
+## Running Appium on Windows
 
 ### Setup
 
 To get started:
 
-1. Install [node.js](http://nodejs.org/download/) (v4 or greater). Use the
-   installer from nodejs.org.
-1. Install `appium-doctor` in order to check your system: `npm install appium-doctor -g`.
-   Then run by typing `appium-doctor`.
-1. Install the [Android SDK](http://developer.android.com/sdk/index.html).
-   You will need to run the 'android' tool (included in the SDK, in the `tools` folder) and make sure
-   you have an API Level 17 or greater API installed. Set `ANDROID_HOME` to be
+   1. Download latest [node and npm tools](https://nodejs.org/download/release/v6.3.0/node-v6.3.0-x64.msi) MSI (version >= 6.0). The `npm` and `nodejs` paths should be in your PATH environment variable. 
```

Comment:
ok, this section isn't exactly what I had in mind. Here's what I was thinking:

```md
### Setup

<put shared setup instructions here>

### Android Setup

<put android-ONLY setup instructions here. this is where the java stuff goes>

### Windows App Testing setup

<put winappdriver-ONLY setup instructions here (like developer mode)>
```

In other words, there's no need to duplicate shared instructions.